Python/chapter01/1.1 - Is Unique/jesus_1.1.sol.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+
+def is_unique(s: str) -> bool:
+    for x in s:
+        if s.count(x) > 1:
+            return False
+    return True
+
+class TestIsUnique(unittest.TestCase):
+
+    def test_is_unique(self):
```

Comment:
this might be okay for now, but for more complex tests, Golang has this wonderful pattern called [table driven tests](https://github.com/golang/go/wiki/TableDrivenTests), where you can restructure the code the following way:

```python
for s, expected in [
  ('aaaaa', False),
  ('abc', True),
  ...
  ('a', True),
]:
    self.assertEqual(is_unique(s), expected, msg=s)
```

to cut down on repetition.

for even further de-duplication, you can split the test in positive and negative tests:

```python
def test_is_unique(self):
  for s in [
    'abc',
    ....
    '',
  ]:
    self.assertTrue(is_unique(s), msg=s)

def test_is_not_unique(self):
  for s in [
    'aaaaa',
    ....
    'abcda',
  ]:
    self.assertFalse(is_unique(s), msg=s)
```